chat-plugins/health-fitness.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Heath and Fitness chat-plugin
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ * 
+ * This handles health and fitness daily challenges as well as a daily quote
+ * 
+ * @license MIT license
+ */
+'use strict';
+
+const FS = require('../fs');
+
+const HF_DATA_PATH = 'config/chat-plugins/health-and-fitness.json';
+const MAX_DATA_LENGTH = 500;
+
+class HealthFitness {
+	constructor() {
+		this.hfData = {
+			cardio: '',
+			gym: '',
+			quote: '',
+			__proto__: null,
```

Comment:
The constructor's not going to be optimizable using `__proto__` in an object literal like this. `Object.create` would probably be a better idea instead, even though this is only getting called once between restarts

Reply:
So, something like this?

```/Javascript
this.hfData = Object.create({
     cardio: '',
     gym: '',
     quote: '',
     __proto__: null,
});
```

Reply:
.No, since that's still using `__proto__`. `Object.assign(Object.create(null), {cardoi: '', gym: '', quote: ''})` allows you to create an object with both a null prototype and properties without needing to mutate `[[Prototype]]`

Reply:
Thanks!

Reply:
Wait, how does the old code mutate `[[Prototype]]`?

Reply:
...you're right actually, I was thinking of when it's used as a setter, not when creating an object.

Reply:
Er, so how should this be done?

Reply:
Your old code was fine. Morfent tends to misunderstand how JavaScript works sometimes. :p Change it back.

Reply:
Okay, I changed it back